test/jobs/maintenance_tasks/task_job_test.rb

```diff
@@ -207,6 +216,16 @@ class TaskJobTest < ActiveJob::TestCase
       assert_predicate @run.reload, :succeeded?
     end
 
+    # Remove this once string cursors are shipped (#339)
+    test '.perform_now handles string cursor when Active Record Relation task resumes' do
+      run = Run.create!(task_name: 'Maintenance::UpdatePostsTask')
+      run.expects(:cursor).returns(Post.pick(:id)&.to_s)
```

Comment:
So you were entirely right about Post fixtures not getting loaded in these tests 😂  Apparently Rails doesn't support loading fixtures from both the engine + the dummy. I'm gonna put up a PR to create some Post records for these tests.

Reply:
@sambostock I attempted to just move the `runs.yml` fixture into the dummy's fixtures and change the path in the test helper, but Mocha went 💥  😅 . I instead decided to go the route of manually creating Posts for any tests that needed them, and actually realized that it's really only this test:
https://github.com/Shopify/maintenance_tasks/blob/4ae6699466cc726dd73906fd3711171534786e63/test/jobs/maintenance_tasks/task_job_test.rb#L219-L226
that needs Posts to be created to be truly testing what we expect. Even our system tests are designed to test interactions with the web interface and actually don't need any Post data to operate on to be considered correct.

I think it just makes sense to create some Posts inline for any test that needs them (this test here and the other Active Record one). We might be able to go straight to #339 anyways, so no need to update this, just an update 😄  (Our OG ActiveRecord test should be fixed, so thanks again for catching this)